pyccl/baryons.py

```diff
@@ -0,0 +1,155 @@
+from . import ccllib as lib
+from .base import CCLAutoreprObject
+from .pyutils import check
+from .pk2d import Pk2D
+from .base import unlock_instance
+import numpy as np
+
+
+class Baryons(CCLAutoreprObject):
```

Comment:
This ought to be an abstract class, so define an `abstractmethod` called `_include_baryonic_effects` which all subclasses should override.

Reply:
Done